recipe/build.sh

```diff
@@ -1,6 +1,12 @@
 set -e
 
-./configure --prefix=$PREFIX --enable-official-khronos-headers
+./bootstrap
+
+./configure \
+  --prefix=$PREFIX \
+  --enable-official-khronos-headers \
+  --enable-custom-vendordir=$PREFIX/etc/OpenCL/vendors
```

Comment:
> Should we create this directory as well?

Maybe. The only point would be that users can find it in order to manually drop in `.icd` files. Otherwise it'll get created once the first ICD is installed.

> Also is there a way to still include the system in the search path here?

That'd be nice. ocl-icd doesn't seem to support multiple search paths when I checked, so it would have to be hacked. For now, users can simply copy the system `.icd` file here and things should just work.